apps/web/screens/Home.tsx

```diff
@@ -19,20 +23,21 @@ import {
   GetLifeEventsQuery,
   QueryGetNewsListArgs,
   QueryGetLifeEventsArgs,
-} from '../graphql/schema'
+} from '@island.is/web/graphql/schema'
 import {
   GET_NAMESPACE_QUERY,
   GET_CATEGORIES_QUERY,
   GET_FRONTPAGE_SLIDES_QUERY,
   GET_NEWS_LIST_QUERY,
   GET_LIFE_EVENTS_QUERY,
 } from './queries'
+import { IntroductionSection } from '@island.is/web/components/IntroductionSection'
+import { LifeEventsCardsSection } from '@island.is/web/components/LifeEventsCardsSection'
+import { Section } from '@island.is/web/components/Section'
```

Comment:
is this imported from '@island.is/web/components' ? if so maybe add this to the '@island.is/web/components' import above